pkg/apis/componentconfig/types.go

```diff
@@ -62,3 +62,213 @@ const (
 	ProxyModeUserspace ProxyMode = "userspace"
 	ProxyModeIPTables  ProxyMode = "iptables"
 )
+
+type KubeletConfiguration struct {
+	// config is the path to the config file or directory of files
+	Config string `json:"config"`
```

Comment:
They match the flag ordering that the flags are added [here](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L195) which is what this was generated from. I can switch to alphabetical although it's sometimes not the best at grouping related flags.